### PR TITLE
Add | ActiveStorage for images

### LINKED ISSUE #14: ActiveStorage for Users
## Description
Users could be update their profile information

## To Do:

- Users should be update photo
- Users should be update name
- Users should be update email
- Users should be update description

### PR DESCRIPTION
**Pull request by OchoaHub**
## Description
Adding activestorage for upload images and modify the model and controller for _User_.

## Test 
- Download the project 
- Run the project
- Register in the app
- Enter to your profile

### CODE REVIEW

**Review by LPMendoza — APPROVED** [review]
Nice code bro! And the changes are the just needed

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class UsersController < Clearance::UsersController
   private
 
   def user_params
-    params.require(:user).permit(:username, :email, :password)
+    params.require(:user).permit(:avatar, :username, :email, :password)
```
Comment (n0b0dy-su) [inline app/controllers/users_controller.rb:6]: Nice, it's important using this ways to validate params

File: Gemfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ ruby '2.7.2'
 gem 'wicked_pdf'
 gem 'wkhtmltopdf-binary'
 gem 'clearance'
-
+gem 'activestorage'
```
Comment (n0b0dy-su) [inline Gemfile:10]: nice Gem for data saving

**Review by n0b0dy-su — APPROVED** [review]

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (LDVera) [inline app/controllers/users_controller.rb:6]: so important use the strong params with the permit, for more security, good job 👍🏼

**Review by LDVera — APPROVED** [review]

**Pull request merged**